lib/stellar_core_commander/commander.rb

```diff
@@ -14,15 +14,23 @@ def initialize(stellar_core_bin)
       @processes = []
     end
 
-    Contract None => Process
-    def make_process
+    Contract String => Process
```

Comment:
Interesting note,  literal values are valid contracts, so you could have expressed this as:

`Or["local", "docker"] => Process`  and the contract system would have behaved as expected.